rclcpp/include/rclcpp/init_options.hpp

```diff
@@ -29,7 +29,7 @@ class InitOptions
 {
 public:
   /// If true, the context will be shutdown on SIGINT by the signal handler (if it was installed).
-  bool shutdown_on_sigint = true;
+  bool shutdown_on_signal = true;
```

Comment:
The comment needs to be updated, as currently this would possible affect SIGTERM too.

But on that point, what if I wanted to shutdown on SIGTERM but not SIGINT? And on another Context shutdown on both.

I think, unfortunately, to do this right, each context needs the ability to shutdown on SIGTERM, SIGINT, both, or neither.

Reply:
We can add that, I'm not sure if it's really useful though.